.babelrc

```diff
@@ -1,4 +1,11 @@
 {
-  "presets": ["es2015"],
-  "ignore": "**/vendor/*"
+  "presets": [
+    ["@babel/preset-env", {
+       "targets": {
+         "browsers": ["> 0.3%"]
```

Comment:
This config means that the (new) JS we write will be compiled to old JS for compatibility with older browsers, but only those that have >= 0.3% market share. You can also explicitly specify particular browser versions that you want to support or specify date ranges (eg. "all versions of Chrome and Firefox that are less than two years old")